addons/mass_mailing/models/mailing.py

```diff
@@ -475,15 +475,13 @@ def create(self, vals_list):
     def write(self, values):
```

Comment:
Small typo in commit message:
"the `ab_testing_completed` filed should not be copied over."
->
"the `ab_testing_completed` field should not be copied over."